ledger-core/idl/address.djinni

```diff
@@ -11,17 +11,5 @@ Address = interface +c {
 
     # Get the currency associated with this address.
     getCurrency(): Currency;
-
-    # Attempt to parse a string address.
```

Comment:
What does all this become?

Reply:
As I said in the description, the static `parse` method - as well as `isValid` - calls underlying coin parse so all this become part of `ledger-core-xxx`.
If these two static methods are required in `ledger-core` library - so far I don't see any usage - we should rethink the way we manage its.